src/content/changes/highlights/2023-03-23.md

```diff
@@ -0,0 +1,21 @@
+---
+date: 2023-03-23T11:00:00
+title: Highlights for the week ending March 23, 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## General
+
+### Observability
+
+* prometheus-agent: cluster releases cointains a new component which enables us to monitor your cluster and provides best quality of service. Some cluster upgrade might require manual intervention from our side, especially if you are using prometheus-operator. Feel free to ask team-atlas for any questions or help needed, also more details like why is this needed and which cluster releases are affected can be found in our doc [here](https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent/).
```

Comment:
@giantswarm/team-teddyfriends just raising awareness on this topic. As customer might experience issue during next cluster upgrade due to us taking ownership of prometheus-operator. Atlas is happy to help during those upgrades.

Reply:
can we define which cluster release version brings this for each provider plz?

Reply:
This is written in the linked doc https://docs.giantswarm.io/getting-started/observability/monitoring/prometheus/agent